lib/player.js

```diff
@@ -4459,6 +4459,12 @@ shaka.Player = class extends shaka.util.FakeEventTarget {
             /* height= */ variant.video.height || NaN);
       }
 
+      if (this.loadMode_ == shaka.Player.LoadMode.SRC_EQUALS) {
```

Comment:
This line is inside the `if (this.loadMode_ == shaka.Player.LoadMode.MEDIA_SOURCE)` clause, that starts at line 4441. So it won't ever be caught, right?
You should move this to be under line 4477.

Reply:
arg...